concordium-sdk/src/main/java/com/concordium/sdk/transactions/AccountAddress.java

```diff
@@ -1,16 +1,22 @@
 package com.concordium.sdk.transactions;
 
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.ToString;
 import lombok.val;
 
+import java.nio.ByteBuffer;
 import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
 import java.util.Objects;
 
 @ToString
+@EqualsAndHashCode
 public final class AccountAddress {
     public static final int BYTES = 32;
     private final static int VERSION = 1;
+    private final static int ACCOUNT_ADDRESS_PREFIX_SIZE = 29;
+    private final static int ALIAS_MAX_VALUE = 2 << 24;
```

Comment:
I don't think this is correct. This is `2^(1 + 24)`.

The max alias is `2^24 - 1`.